types/ckeditor__ckeditor5-build-balloon-block/index.d.ts

```diff
@@ -0,0 +1,101 @@
+import EasyImage from '@ckeditor/ckeditor5-easy-image/src/easyimage';
+import Heading from '@ckeditor/ckeditor5-heading/src/heading';
+import Image from '@ckeditor/ckeditor5-image/src/image';
+import ImageCaption from '@ckeditor/ckeditor5-image/src/imagecaption';
+import ImageStyle from '@ckeditor/ckeditor5-image/src/imagestyle';
+import ImageToolbar from '@ckeditor/ckeditor5-image/src/imagetoolbar';
+import ImageUpload from '@ckeditor/ckeditor5-image/src/imageupload';
+import Indent from '@ckeditor/ckeditor5-indent/src/indent';
+import Link from '@ckeditor/ckeditor5-link/src/link';
+import List from '@ckeditor/ckeditor5-list/src/list';
+import MediaEmbed from '@ckeditor/ckeditor5-media-embed/src/mediaembed';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import PasteFromOffice from '@ckeditor/ckeditor5-paste-from-office/src/pastefromoffice';
+import Table from '@ckeditor/ckeditor5-table/src/table';
+import TableToolbar from '@ckeditor/ckeditor5-table/src/tabletoolbar';
+import TextTransformation from '@ckeditor/ckeditor5-typing/src/texttransformation';
+import CloudServices from '@ckeditor/ckeditor5-cloud-services/src/cloudservices';
+
+export default class BalloonEditor extends BalloonEditorBase {
+    static builtinPlugins: [
+        typeof Essentials,
+        typeof UploadAdapter,
+        typeof Autoformat,
+        typeof BlockToolbar,
+        typeof Bold,
+        typeof Italic,
+        typeof BlockQuote,
+        typeof CKFinder,
+        typeof CloudServices,
+        typeof EasyImage,
+        typeof Heading,
+        typeof Image,
+        typeof ImageCaption,
+        typeof ImageStyle,
+        typeof ImageToolbar,
+        typeof ImageUpload,
+        typeof Indent,
+        typeof Link,
+        typeof List,
+        typeof MediaEmbed,
+        typeof Paragraph,
+        typeof PasteFromOffice,
+        typeof Table,
+        typeof TableToolbar,
+        typeof TextTransformation,
+    ];
+
+    static defaultConfig: {
+        blockToolbar: [
+            'heading',
+            '|',
+            'bulletedList',
+            'numberedList',
+            '|',
+            'outdent',
+            'indent',
+            '|',
+            'uploadImage',
+            'blockQuote',
+            'insertTable',
+            'mediaEmbed',
+            '|',
+            'undo',
+            'redo',
+        ];
+        toolbar: {
+            items: ['bold', 'italic', 'link'];
+        };
+        image: {
+            toolbar: [
+                'imageStyle:inline',
+                'imageStyle:block',
+                'imageStyle:side',
+                '|',
+                'toggleImageCaption',
+                'imageTextAlternative',
+            ];
+        };
+        table: {
+            contentToolbar: ['tableColumn', 'tableRow', 'mergeTableCells'];
+        };
+        // This value must be kept in sync with the language defined in webpack.config.js.
```

Comment:
```ts
        // This value must be kept in sync with the language defined in webpack.config.js.
```
does it mean this is not a contant and should be just a `string`?
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-build-balloon-block/webpack.config.js#L51

Reply:
Agreed that `string` seems more maintainable and isn't obviously less useful.